modules/rosa-deleting-aws-resources-aws-console.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa-sts-deleting-cluster.adoc
+
+[id="rosa-deleting-aws-resources-aws-console_{context}"]
+= Deleting the AWS resources by using the AWS IAM Console
+
+After deleting a {product-title} (ROSA) cluster, you can delete the AWS Security Token Service (STS) resources by using the AWS Identity and Access Management (IAM) Console.
+
+[IMPORTANT]
+====
+Account-wide IAM roles and policies might be used by other ROSA clusters in the same account. You must only remove the resources if they are not required by other clusters.
+====
+
+.Prerequisites
+
+* You have deleted your ROSA cluster. For more information, see the _Deleting a cluster_ section.
+
+.Procedure
+
+. Log in to the link:https://console.aws.amazon.com/iamv2/home#/home[AWS IAM Console].
+
+. Delete the OpenID Connect (OIDC) provider that you created for Operator authentication in your cluster:
+.. Navigate to *Access management* -> *Identity providers* and click on the OIDC resource that you created to authenticate the cluster Operators.
+.. In the dialog page for the resource, select *Delete* to delete the OIDC provider.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[IMPORTANT]
+====
+You must delete the cluster before you remove the Operator roles.
+====
++
+[TIP]
+====
+The IAM role and policy ARNs include the role prefix that is specified when the STS resources are created. The default prefix is `ManagedOpenShift`.
+====
++
+.. Navigate to *Access management* -> *Roles* and click on one of the cluster-specific Operator roles that you created for your cluster.
+.. In the dialog page for the resource, select *Delete role* to delete the role. Select *Yes, delete* to confirm the role deletion.
```

Comment:
There should be also `Select Yes, delete to confirm the role deletion.` for `Delete the OpenID Connect (OIDC) provider`